Source/Krypton Toolkit/Examples/AdvancedDataGridView.cs

```diff
@@ -1,43 +1,40 @@
-﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel;
+﻿using System.ComponentModel;
 using System.Data;
-using System.Drawing;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using System.Windows.Forms;
 
 using Krypton.Toolkit.Suite.Extended.AdvancedDataGridView;
 
 using Timer = System.Windows.Forms.Timer;
+// ReSharper disable AssignNullToNotNullAttribute
+#pragma warning disable CS8622
```

Comment:
I'm not a fan of these !
Rather have them visible in the build and then solve later